`dbt show` only supports selecting a single node (#7848)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/show.md

---

Use `dbt show` to:
- Compile the dbt-SQL definition of a single `model`, `test`, `analysis`, or an arbitrary dbt-SQL query passed `--inline`
  - `dbt show` does not support [Python (dbt-py)](/docs/build/python-models) models.

>   - only selecting a single node is supported so [selector methods](/reference/node-selection/methods), [graph operators](https://docs.getdbt.com/reference/node-selection/graph-operators), etc. that select multiple nodes will not be utilized

Reviewer: ```suggestion
  - Only selecting a single node is supported. [Selector methods](/reference/node-selection/methods), [graph operators](/reference/node-selection/graph-operators), and other methods that select multiple nodes cannot be used.
```

Author: These methods _can_ be used -- it just won't take multiple nodes into account.

So you can use whichever methods you want to determine the selection, and it will only use a maximum of one node.

Author: ```suggestion
  - Only selecting a single node is supported. [Selector methods](/reference/node-selection/methods), [graph operators](/reference/node-selection/graph-operators), and other methods that select multiple nodes will not be utilized.
```
